src/components/note-card/NoteCard.jsx

```diff
@@ -0,0 +1,40 @@
+import {
+  BsPin,
+  MdArchive,
+  FaTrash,
+  MdOutlineEdit,
+  MdLabel,
+} from "assets/icons/icons";
+import "./NoteCard.css";
+
+const NoteCard = ({ note }) => {
+  var today = new Date();
```

Comment:
suggestion - instead of time you can use currentTime or another proper variable name.